system_metrics_collector/test/moving_average_statistics/test_moving_average_statistics.cpp

```diff
@@ -181,6 +181,16 @@ TEST_F(MovingAverageStatisticsTestFixture, test_thread_safe) {
   ASSERT_NEAR(moving_average_statistics->average(), control, var);
 }
 
+TEST(MovingAverageStatisticsTest, test_pretty_printing) {
+  StatisticData data;
+  ASSERT_EQ("avg=nan, min=nan, max=nan, std_dev=nan, count=0", statisticsDataToString(data));
+
+  MovingAverageStatistics stats;
+  stats.addMeasurement(1);
+  ASSERT_EQ("avg=1.000000, min=1.000000, max=1.000000, std_dev=0.000000, count=1",
+    statisticsDataToString(stats.getStatistics()));
```

Comment:
I think this comment was not sent last time somehow - be careful with exact matches, the precision is probably platform dependent, as well as how they are displayed.